src/Fable.Transforms/Dart/Replacements.fs

```diff
@@ -3145,7 +3145,7 @@ let results
     (args: Expr list)
     =
     match i.CompiledName with
-    | ("Bind" | "Map" | "MapError" | "IsOk" | "IsError") as meth ->
+    | ("Bind" | "Map" | "MapError" | "IsOk" | "IsError" | "Contains" | "Count" | "DefaultValue" | "DefaultWith" | "Exists" | "Fold" | "FoldBack" | "ForAll" | "Iterate" | "ToArray" | "ToList" | "ToOption") as meth ->
```

Comment:
If we remove the parenthesis it seems like Fantomas is able to format the code to something like that:

```fs
    match i.CompiledName with
    | "Bind"
    | "Map"
    | "MapError"
    | "IsOk"
    | "IsError"
    | "Contains"
    | "Count"
    | "DefaultValue"
    | "DefaultWith"
    | "Exists"
    | "Fold"
    | "FoldBack"
    | "Iterate"
    | "ToArray"
    | "ToList"
    | "ToOption" as meth -> ...
```

Perhaps we should do that as I feel it is easier to read than having a really long line?